### PR TITLE
Change dev dependencies dependabot updates frequency

### PR DESCRIPTION
**Pull request by NSeydoux**

### CODE REVIEW

**Comment by ThisIsMissEm** [comment]
I've revoked the token used, see our internal chat for details.

File: .github/workflows/dependabot-automation.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (ThisIsMissEm) [inline .github/workflows/dependabot-automation.yml:16]: I think you can use the github actions access token here

**Comment by NSeydoux** [comment]
See https://github.com/inrupt/temp-test-dependabot-automation for experimentations showing the config looks correct.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,23 +1,49 @@
...
-      interval: "daily"
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "pip"
   # Enable version updates for our CI tooling
   - package-ecosystem: "github-actions"
     # Look for `package.json` and `lock` files in the `root` directory
     directory: "/"
     # Check the npm registry for updates every day (weekdays)
```
Comment (chelseapinka) [inline .github/dependabot.yml:44]: Should we update this comment to reflect that is is now checking weekly? Also we could add which day of the week we believe the schedule will be triggered on

File: .github/dependabot.yml [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+      - "npm"
+      - "development"
+    allow:
+      - dependency-type: development
+
   # Enable version updates for the website tooling
   - package-ecosystem: "pip"
     # Look for `package.json` and `lock` files in the `root` directory
     directory: "/docs"
     # Check the npm registry for updates every day (weekdays)
```
Comment (chelseapinka) [inline .github/dependabot.yml:34]: Should we update the comment to reflect the weekly checks?

File: .github/dependabot.yml [hunk 4]
```diff
@@ -1,23 +1,49 @@
...
+    labels:
+      - "dependencies"
+      - "npm"
+      - "development"
+    allow:
+      - dependency-type: development
+
   # Enable version updates for the website tooling
   - package-ecosystem: "pip"
     # Look for `package.json` and `lock` files in the `root` directory
```
Comment (chelseapinka) [inline .github/dependabot.yml:32]: update `root` in comment or remove comment entirely?

Comment (NSeydoux) [reply .github/dependabot.yml]: Removed in 7b2df5c22ca51381750df223637b8bd1285ae8aa

File: .github/dependabot.yml (hunk 3 above)
Comment (NSeydoux) [reply .github/dependabot.yml]: Fixed in d15f9ee31a2cb09927a87efab780ec6b0a96e478

**Review by ThisIsMissEm — COMMENTED** [review]
The changes here look reasonable, and inline with what we would want to adopt. However, the pull request title is incorrect, and we probably need to discuss in parking lot next steps on this PR / Ticket.

**Review by ThisIsMissEm — APPROVED** [review]

**Comment by ThisIsMissEm** [comment]
Looks like this needs to be rebased against `main` though :(

**Pull request merged**